skafka/src/main/scala/com/evolutiongaming/skafka/consumer/RebalanceListener1.scala

```diff
@@ -61,8 +61,48 @@ trait RebalanceListener1[F[_]] {
 
 }
 
+/**
+  * Same as [[RebalanceListener1]] but with a `consumer` to allow a better type inference.
+  *
+  * {{{
+  *    import RebalanceCallback.implicits._ // to allow writing `someF.lift` instead of `lift(someF)`
+  *
+  *    def onPartitionsRevoked(partitions: Nes[TopicPartition]) = {
+  *      groupByTopic(partitions) traverse_ {
+  *        case (_, partitions) =>
+  *          for {
+  *            _ <- someF.lift
+  *            partitionsOffsets <- partitions.toNonEmptyList traverse { partition =>
+  *              // fails to compile with `RebalanceCallback.position` variant at
+  *              // _ <- someF2(partitionsOffsets).lift
+  *              // expected type RebalanceCallback[Nothing,?] but found RebalanceCallback[F,Unit]
+  *              consumer.position(partition) map (partition -> _)
+  *            }
+  *            _ <- someF2(partitionsOffsets).lift
+  *          } yield ()
+  *      }
+  *    }
+  *    def someF: F[Unit] = ???
+  *    def someF2(a: Any): F[Unit] = ???
+  * }}}
+  */
+trait RebalanceListener1WithConsumer[F[_]] extends RebalanceListener1[F] {
```

Comment:
I propose to move this class under `RebalanceListener1`

Reply:
to `RebalanceListener1` companion object?

please note that it's already located in the file together with main trait `RebalanceListener1`
src/main/scala/com/evolutiongaming/skafka/consumer/RebalanceListener1.scala